Core/SupportedExternalURLScheme.swift

```diff
@@ -32,10 +32,15 @@ public struct SupportedExternalURLScheme {
         "itms-appss",
         "itunes"
     ]
-
+    
     public static func isSupported(url: URL) -> Bool {
         guard let scheme = url.scheme else { return false }
-        return schemes.contains(scheme)
+        return defaultSchemes.contains(scheme)
     }
-
+    
+    public static func isProhibited(url: URL) -> Bool {
+        guard let scheme = url.scheme else { return true }
+        return scheme == "about"
```

Comment:
Can we follow a similar pattern to `defaultSchemes` here?

Reply:
done